app/kuma-cp/cmd/root.go

```diff
@@ -41,6 +50,10 @@ func newRootCmd() *cobra.Command {
 	}
 	// root flags
 	cmd.PersistentFlags().StringVar(&args.logLevel, "log-level", kuma_log.InfoLevel.String(), kuma_cmd.UsageOptions("log level", kuma_log.OffLevel, kuma_log.InfoLevel, kuma_log.DebugLevel))
+	cmd.PersistentFlags().StringVar(&args.outputPath, "log-output-path", args.outputPath, "path of output rotating file Example: if we set it to /tmp/kuma.log then after the file is rotated we will have /tmp/kuma-2021-06-07T09-15-18.265.log")
```

Comment:
`path of output rotating file Example...` -> `path to the file that will be filled with logs. Example...`